server/instance.go

```diff
@@ -357,6 +357,20 @@ func (s *Server) registerInstanceRoutes(g *echo.Group) {
 		}
 		entry := list[0]
 
+		var lastRecordedSchema string
+		// For baseline migration, find its prev migration history
```

Comment:
Comment the reason why we only need to do this for baseline migration.

Actually, I am thinking it's also worth returning the prev migration history for normal migraiton.

So if prevSchema and lastRecordedSchema is different, then we can signal there exists a drift when applying the new migration.

And this helps because otherwise user would be confused by the UI info since the equation doesn't meet:

schemaPrev + Migration change = schema

Reply:
good idea!